docker/Dockerfile

```diff
@@ -30,6 +36,7 @@ RUN \
       which \
       cyrus-sasl-devel && \
     # Install libmicrohttpd from source
+    cd /opt && \
```

Comment:
Why the added `cd /opt &&` steps? Is some kind of fixing in the Dockerfile?

Reply:
The directrices of the source codes are nested if the 'cd /opt' doesn't be added.
The source code of the Orion is as shown:
```
/opt/libmicrohttpd-0.9.48/mongo-cxx-driver-legacy-1.1.2/fiware-orion
```

Reply:
Ok. Thanks for the fix!

NTC